packages/@aws-cdk-testing/cli-integ/resources/cli-regression-patches/v2.142.0/skip-tests.txt

```diff
@@ -0,0 +1,4 @@
+# Skipping the test to fix issue https://github.com/aws/aws-cdk/issues/30241. 
+# cli-integ tests failing for the old tests with the new cli changes for list stacks.
```

Comment:
`cli-integ tests failing for the old tests with the new cli changes for list stacks.` Should that line be removed?

Reply:
I think just the line above is sufficient and true, whereas that line isn't true?

Reply:
No, I've cleaned it up but, these are just comments I've added. The test we'd want to skip is `cdk diff picks up changes that are only present in changeset`